targets/targets.json

```diff
@@ -635,6 +635,23 @@
             "network-default-interface-type": "ETHERNET"
         }
     },
+    "SDT64B": {
+        "core": "Cortex-M4F",
```

Comment:
why this target do not inherit from MCU K64F ? IT looks like it would benefit and override few details .

Reply:
Similar to other targets below.

Reply:
I will simplify "SDT64B", but we cannot simplify the codes for the other boards. I will explain the reason in the next comment.

Reply:
Thanks, please do !

Ideally, K64F and some other targets contain base target (MCU_xxx) that we can inherit from. So if there are targets that do not have this hierarchy, they do not allow us to inherit from and set-up own target